src/tiles/list/tile-list.ts

```diff
@@ -292,6 +292,19 @@ export class TileList extends LitElement {
     );
   }
 
+  private get textSnippetsTemplate() {
+    if (!this.hasSnippets) return nothing;
+
+    return html`
+      <text-snippet-block .viewSize=${'list'}></text-snippet-block>
```

Comment:
```suggestion
      <text-snippet-block viewSize="list"></text-snippet-block>
```
I want to say writing it as attribute removes the linter's bark this need to interpolate `=${'list'}`

Reply:
👍